src/components/myprofile/MyProfile.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './MyProfile.css';
+
+const MyProfile = () => {
+  const rocketsState = useSelector((state) => state.rockets.rockets);
+  const missionsState = useSelector((state) => state.missions.missions);
+
```

Comment:
- [ ] Also, Since you fetch data from the endpoint when we switch between pages, the reservation for mission and rocket keeps being replaced on the page switch. As you can see, once I reserve a rocker and admission I can see the reservation in the profile section. But once I go back to rocket or mission page the reservation is not available, we need to keep reservations between page switches.

### current behavior 

https://github.com/DevKaranJ/space-travelers-hub/assets/69077061/b955a4da-ab5f-4e26-826a-d5e46d17610d

Reply:
- Well resolved